test/Microsoft.Framework.DesignTimeHost.Tests/TestAssemblyLoadContext.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using Microsoft.Framework.Runtime;
+
+namespace Microsoft.Framework.DesignTimeHost
+{
+    public class TestAssemblyLoadContext : IAssemblyLoadContext
+    {
+        private readonly IReadOnlyDictionary<string, Assembly> _assemblyNameLookups;
+
+        public TestAssemblyLoadContext(IReadOnlyDictionary<string, Assembly> assemblyNameLookups)
+        {
+            _assemblyNameLookups = assemblyNameLookups;
+        }
+
+        public void Dispose()
+        {
+            throw new NotImplementedException();
```

Comment:
nit: remove this line to avoid exceptions if framework ever cleans up `IDisposable` contexts